wp-content/themes/pub/wporg-learn-2024/patterns/archive-lessons-content.php

```diff
@@ -10,13 +10,23 @@
 <!-- wp:group {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|50"}}},"layout":{"type":"constrained","justifyContent":"left","contentSize":"730px"}} -->
 <div class="wp-block-group" style="margin-bottom:var(--wp--preset--spacing--50)">
 
+	<!-- wp:query-title {"type":"archive","showPrefix":false} /-->
+
 	<!-- wp:paragraph -->
-	<p>Introduction TBC.</p>
+	<p><?php esc_html_e( 'Introduction TBC.', 'wporg-learn' ); ?></p>
```

Comment:
Is this theme synced into GlotPress yet? If so, this should stay as a plain string until you know what it is, otherwise polyglots might think they need to translate it.

Reply:
I was holding off wrapping them until they were final, but then I realized there was the GlotPress step. I'm not fully aware of how that works though, how can I check?

Reply:
The code for the syncing is in `website/translate/bin/` in dotorg trac, but I find it pretty complicated to follow. However, you can also check in #polyglots in slack, and see that there have been strings changing in learn:

<img width="607" alt="Screenshot 2024-06-04 at 6 46 56 PM" src="https://github.com/WordPress/Learn/assets/541093/dcd4db03-e649-4bdc-9a27-68331de7053b">

And if you click into one of the locales, and click details on the strings, you can see they're already being pulled from `pub/wporg-learn-2024`

<img width="1633" alt="Screenshot 2024-06-04 at 6 48 18 PM" src="https://github.com/WordPress/Learn/assets/541093/83c560cb-f3c8-4075-9e8f-8a2a1ed2e4ca">

So it is already syncing.

Reply:
Oh no, I should have checked there. I'll clean up asap.

Reply:
Cleaned up in https://github.com/WordPress/Learn/compare/1cee272c09f8...5750e0ffa5e4